docs/discover/search-sessions.asciidoc

```diff
@@ -1,20 +1,19 @@
 [[search-sessions]]
 == Run a search session in the background
 
-Sometimes you might need to search through large amounts of data no matter
```

Comment:
just a thought about that, since search sessions are a general concept it might be a good fit for **Production considerations**